index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
According to Mate guides you should add empty lines between siblings.
Check all your html file and add empty lines where needed.
```html
<body>
  <div>
    <span>1</span>
  
    <p>2</p>
  </div>

  <div>
    <div>
      <button></button>  
    </div>

    <span></span>
  </div>
</body>
```

But there are exceptions for example when you have a bunch of repeated tags in container like here:
```html
<ul>
  <li><a></a></li>
  <li><a></a></li>
  <li><a></a></li>
  <li><a></a></li>
</ul>

<ul>
  <li></li>
  <li></li>
  <li></li>
  <li></li>
</ul>
```

Also as Darek mentioned you should fix formatting in few places. When tag hase more than 2 attributes for readability  you should format them like so:
```html

//BAD example:
          <img class="phone__image" src="./src/images/icon-phone.svg"
            alt="Phone">


//Good example
          <img 
              class="phone__image" 
              src="./src/images/icon-phone.svg"
              alt="Phone"
            >
```

Revalidate the whole index.html file as those rules makes your code more readable and it is also easier for reviewers to check your code :)